### DISCUSSION TITLE
Discord bot

### DISCUSSION THREAD

**Discussion by MaryamAK189** [PlutoyDev/sky-shards#67] [General]
Do you plan on ever making this into a discord bot that notifies you when the shards land?

**Comment by PlutoyDev** [comment]
If I recalled correctly there has been a discord bot made. It combined with other timings, reminders, etc but I can't seem to find it.
Personally I won't making a discord bot for this primarily due to my lack of time day to day. secondly due to bot hosting cost money which I don't intend to pay on top of how much I'm paying to host other stuff.

Thank you for the suggestion
